packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -307,7 +308,10 @@ export class DocumentDeltaConnection
 		return this.details.initialClients;
 	}
 
-	protected emitMessages(type: string, messages: IDocumentMessage[][]) {
+	protected emitMessages(
+		type: string,
+		messages: (IDocumentMessage | unknown)[][] | ISentSignalMessage[],
```

Comment:
should this be 
```suggestion
		messages: (IDocumentMessage | ISentSignalMessage)[][],
```

Reply:
Chatted offline. I will undo the typing changes in this PR and follow up with another PR in `next` to correct the signal typings since those will be breaking changes.